lib/events/athena/test.go

```diff
@@ -169,6 +169,13 @@ func SetupAthenaContext(t *testing.T, ctx context.Context, cfg AthenaContextConf
 	if ok, _ := strconv.ParseBool(testEnabled); !ok {
 		t.Skip("Skipping AWS-dependent test suite.")
 	}
+	var bypassSNS bool
+	if s := os.Getenv(teleport.AWSRunTests + "_ATHENA_BYPASS_SNS"); s != "" {
+		if ok, _ := strconv.ParseBool(s); ok {
+			t.Log("bypassing SNS for Athena audit log events")
+			bypassSNS = true
+		}
+	}
```

Comment:
nit: I would recommend to not the flow behind other additional env flag. 

What would you say about just adding the bool flag to `SetupAthenaContext(SetupAthenaContext{bypassSNS:true}) `

and copy and paste some basic Athena test ?